### PR TITLE
New traitor item: storage implant

### PR DESCRIPTION
**Pull request by Core0verload**
It costs you 8TC and allows you to store up to two box-sized items in it. How? Bluespace!
- You can place items in or remove them at any time.
- Placing items in is silent.
- You can inject two implants at once. This will make your storage two times bigger.
- Surgically removing storage implant will drop all it's items.
- No, it can't store nuclear auth disk.

Other syndie storages changes:
- Placing items in syndie dufflebag or syndie toolbox is silent too.
- Syndie dufflebags are illegal 1 in RnD.
- Red insulated gloves are replaced with combat gloves. They are insulated and fireproof.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 1]
```diff
@@ -82,7 +83,7 @@
 	new /obj/item/weapon/crowbar/red(src)
 	new /obj/item/weapon/wirecutters(src, "red")
 	new /obj/item/device/multitool(src)
-	new /obj/item/clothing/gloves/color/red/insulated(src)
```
Comment (Anonus) [inline code/game/objects/items/weapons/storage/toolbox.dm]: Don't, though, I've got a pull to change the red gloves sprite to be a bit more, ah, red and less pale and shitty.

Comment (Anonus) [inline code/game/objects/items/weapons/storage/toolbox.dm]: At least make it red combat gloves or something.

**Comment by bgobandit** [comment]
This makes the smuggler satchel 100% obsolete, can it just be removed already

File: code/game/objects/items/weapons/storage/toolbox.dm (hunk 1 above)
Comment (Core0verload) [inline code/game/objects/items/weapons/storage/toolbox.dm]: Syndicate's corporate colors are black and red. Combat gloves are black, and they look better even after sprite changes.

**Comment by Core0verload** [comment]
@bgobandit, how about making smuggler satchel subtype of a syndie dufflebag? Then it would be useful at least as a bigger backpack without slowdown downsides.

**Comment by SconesC** [comment]
@bgobandit Satchel is/was only ever used for: bombings OR hiding an objective til you want to grab it

**Comment by PKPenguin321** [comment]
How do you put things in/take things out of the implant? Do you have, like, a retractable tray that you just vomit out?

**Comment by Core0verload** [comment]
It works like a regular storage that opens on UI button press. Fluff-wize, you open a BoH-like bluespace pocket on your body.

**Comment by bgobandit** [comment]
there are too many traitor items in general, ist he thing

File: code/game/objects/items/weapons/storage/toolbox.dm (hunk 1 above)
Comment (Anonus) [inline code/game/objects/items/weapons/storage/toolbox.dm]: Yes, but black is boring and common.

**Comment by PKPenguin321** [comment]
that's a stupid reason to not want this bgo

**Comment by bgobandit** [comment]
no, I'm saying get rid of smuggler satchel in favor of this

**Comment by PKPenguin321** [comment]
smuggler satchel would still have it's uses after this, even if it wouldnt have as many

why bother removing it for essentially no reason

**Comment by bgobandit** [comment]
I already fucking said

there are way too many traitor items and it is honestly getting ridiculous at this point

**Comment by Durkel** [comment]
Variety is the spice of life.

That and people are going to meta the combat gloves, they already do in ops.

**Comment by Xhuis** [comment]
Seems like a good idea, but it needs to be openable by a third party (remove the implant and the items drop?) and also not cause issues with traitor objectives failing if something is stored in there.

**Comment by Robustin** [comment]
After learning that you can recolor the red gloves using crayons back to traditional yellow, I errr, support keeping them red now.

Signed,

Stealth NukeOp "Station Engineer"

Also if you're worried about too many items get rid of chameleon stamp, playing cards, etc. This actually has a purpose and I could see having fun with getting stuff past security with it.

**Comment by Alek2ander** [comment]
Security really needs some kind of implant detection at this point.

**Comment by Core0verload** [comment]
Added changelog. This is ready to merge now.

**Comment by Anonus** [comment]
They probably don't need combat gloves. All they do is let you remove lightbulbs, and they're not even red.

**Comment by optimumtact** [comment]
Security doesn't need implant detection

**Pull request merged**